airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1198,6 +1198,14 @@ def __init__(
         impersonation_chain: str | Sequence[str] | None = None,
         **kwargs,
     ) -> None:
+        if bigquery_conn_id:
+            warnings.warn(
+                "The bigquery_conn_id parameter has been deprecated. Use the gcp_conn_id parameter instead.",
+                AirflowProviderDeprecationWarning,
+                stacklevel=2,
+            )
+            gcp_conn_id = bigquery_conn_id
+        
```

Comment:
This operator is deprecated for almost 3 years.

I prefer to start a breaking change release and remove old deprecations rather than start handling with fixing old operators.

Reply:
cc @VladaZakharova please take consider this. We did the same for Amazon provider and i think it really helped users to avoid using deprecated code. The google provider has a lot of deprecations.